src/Card/Card.tsx

```diff
@@ -119,7 +119,7 @@ const Container = styled("div")(({ theme }) => ({
   },
 }))
 
-const Content = styled("div")<{ fullSize?: boolean }>`
+const Content = styled("div", { shouldForwardProp: prop => prop !== "fullSize" })<{ fullSize?: boolean }>`
```

Comment:
```suggestion
const Content = styled.div<{ fullSize?: boolean }>`
```

this should work as well